share/rpcauth/README.md

```diff
@@ -15,4 +15,5 @@ positional arguments:
 
 optional arguments:
   -h, --help  show this help message and exit
+  -j, -json   output data in json format
```

Comment:
I disagree with `-json`, should be `--json`, same as `--help` (not `-help`).

Reply:
It makes sense to go with -j / --json for me as well.

Reply:
Here's an example:
```diff
diff --git a/share/rpcauth/rpcauth.py b/share/rpcauth/rpcauth.py
index 5dd49c90b8..a40e981502 100755
--- a/share/rpcauth/rpcauth.py
+++ b/share/rpcauth/rpcauth.py
@@ -25,7 +25,7 @@ def main():
     parser = ArgumentParser(description='Create login credentials for a JSON-RPC user')
     parser.add_argument('username', help='the username for authentication')
     parser.add_argument('password', help='leave empty to generate a random password or specify "-" to prompt for password', nargs='?')
-    parser.add_argument("-json", help="output to json instead of plain-text", action='store_true')
+    parser.add_argument("-j", "--json", help="output to json instead of plain-text", action='store_true')
     args = parser.parse_args()
 
     if not args.password:
```